src/angular/header-lean/header.scss

```diff
@@ -32,6 +32,14 @@
   }
 }
 
+.sbb-header-icon-actions {
+  white-space: nowrap;
+
+  .sbb-frameless-button:not(:last-child) {
```

Comment:
Does this need to be this specific to `.sbb-frameless-button`?
I feel like we could maybe use `flex` with `gap` here.
What do you think?

Reply:
Totally makes sense. Thanks!